src/service.ts

```diff
@@ -152,10 +154,17 @@ class ServiceWorker {
       if (e.data.success) {
         cb.fn(e);
       } else {
+        this.workerErrorCallback = null;
         cb.ex(e);
       }
       this.workerCallbacks[e.data.id] = null;
     });
+    this.worker.addEventListener("error", (e) => {
```

Comment:
This shall never happen, unless there some critical error with worker. From the descriptions of issue and PR, I cannot understand what causes it and why we cannot fix it on worker's processMessage side.